configs/setups/awiesm/awiesm.yaml

```diff
@@ -183,7 +183,12 @@ echam:
                 OIFS_OASIS_INCLUDE: '"-I$OIFS_OASIS_BASE/build/lib/psmile -I$OIFS_OASIS_BASE/build/lib/psmile/scrip -I$OIFS_OASIS_BASE/build/lib/psmile/mct -I$OIFS_OASIS_BASE/build/lib/psmile/mct/mpeu"'
                 OASIS3MCT_FC_LIB: "$(pwd)/lib/"
                 OASIS3MCTROOT: "$(pwd)/oasis/"
-                configure_opts: '--with-coupler=oasis3-mct'
+                configure_opts: "'--with-coupler=oasis3-mct --without-regard-for-quality'"
+            # Unset FFLAGS, they are set in mh-linux in echam (booo this is horrible)
```

Comment:
@Paul XD made me laugh really hard. I'm okay with leaving it like this, I just don't understand why we need to set them on on the echam file and here off.

Reply:
Hmmm. I think only for albedo for now. Since the other machines work fine, I would avoid tampering with it.